### PR TITLE
groonga 5.0.5

### PR DESCRIPTION
**Pull request by cosmo0920**

### CODE REVIEW

File: Library/Formula/groonga.rb [hunk 1]
```diff
@@ -18,9 +20,9 @@ class Groonga < Formula
   depends_on "lz4" => :optional
   depends_on "openssl"
 
-  depends_on "glib" if build.include? "enable-benchmark"
+  depends_on "glib" if build.with? "benchmark"
 
-  option "enable-benchmark", "Enable benchmark program for developer use"
+  deprecated_option "enable-banchmark" => "with-benchmark"
```
Comment (DomT4) [inline Library/Formula/groonga.rb]: There's a tiny typo here, `enable-banchmark` instead of `enable-benchmark`.

Otherwise :+1: :heart:

Comment (cosmo0920) [inline Library/Formula/groonga.rb]: Thanks!! I didn't notice about it.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/209cafe81c4e556733e87e0da838e94c6878c579. Thank you @cosmo0920!

**Comment by cosmo0920** [comment]
Thanks, @DomT4 ! :smile:

**Pull request closed without merging**